common/common-interactor/src/interactor_helpers.rs

```diff
@@ -568,11 +581,16 @@ pub trait InteractorHelpers {
             amount,
             fee,
             with_transfer_data,
-            is_sovereign_token,
             is_execute,
             expected_error,
         } = bcc;
 
+        let is_sovereign_token = if token.is_some() {
+            token.clone().unwrap().token_id.split('-').nth(0) == Some("SOV")
+        } else {
+            false
```

Comment:
Something doesn't look good here :thinking: 
`nth(0)` for a sovereign token should be the prefix `sov`